Aptoide/appcoins-wallet-android
# [MOB-791] Implemented page view events to Rakam;

**What does this PR do?**

This PR implements page view events to Rakam across the app;

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] BasePageViewFragment.kt

**How should this be manually tested?**

Check all screens you can think of in the app and verify the event is being sent. You can filter logcat by "rakam" to see the specific event logger that logs these events.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-791](https://aptoide.atlassian.net/browse/MOB-791)

**Code Review Checklist**
